ehsm_kms_service/function.js

```diff
@@ -264,8 +264,10 @@ const enroll_user_info = (action, DB, res, req) => {
   if (napi_res) {
     const { appid, apikey } = napi_res.result
     let cmk_res = napi_result(cryptographic_apis.CreateKey, res, [0, 0])
+    let sm_default_cmk_res = napi_result(cryptographic_apis.CreateKey, res, [0, 0])

```

Comment:
will the result be checked?

Reply:
Yes, it has been tested

Reply:
generally, the abbreviation for "result" is "ret" not "res"!

anyway, you can change it later.